--- old.conf	2020-09-26 18:57:58.531490209 -0400
+++ weechat.conf	2020-09-26 11:50:54.000000000 -0400
@@ -34,14 +34,15 @@
 buffer_search_force_default = off
 buffer_search_regex = off
 buffer_search_where = prefix_message
-buffer_time_format = "%H:%M:%S"
+buffer_time_format = "%H:%M"
+buffer_time_same = ""
 color_basic_force_bold = off
 color_inactive_buffer = on
 color_inactive_message = on
 color_inactive_prefix = on
 color_inactive_prefix_buffer = on
 color_inactive_time = off
-color_inactive_window = on
+color_inactive_window = off
 color_nick_offline = off
 color_pairs_auto_reset = 5
 color_real_white = off
@@ -57,7 +58,7 @@
 highlight = ""
 highlight_regex = ""
 highlight_tags = ""
-hotlist_add_conditions = "${away} || ${buffer.num_displayed} == 0 || ${info:relay_client_count,weechat,connected} > 0"
+hotlist_add_conditions = "${away} || ${buffer.num_displayed} == 0"
 hotlist_buffer_separator = ", "
 hotlist_count_max = 2
 hotlist_count_min_msg = 2
@@ -85,10 +86,11 @@
 jump_smart_back_to_buffer = on
 key_bind_safe = on
 key_grab_delay = 800
-mouse = off
+mouse = on
 mouse_timer_delay = 100
 nick_color_force = ""
 nick_color_hash = djb2
+nick_color_hash_salt = ""
 nick_color_stop_chars = "_|["
 nick_prefix = ""
 nick_suffix = ""
@@ -98,7 +100,7 @@
 paste_max_lines = 1
 prefix_action = " *"
 prefix_align = right
-prefix_align_max = 0
+prefix_align_max = 15
 prefix_align_min = 0
 prefix_align_more = "+"
 prefix_align_more_after = on
@@ -111,10 +113,11 @@
 prefix_network = "--"
 prefix_quit = "<--"
 prefix_same_nick = ""
-prefix_suffix = "|"
+prefix_same_nick_middle = ""
+prefix_suffix = "¦"
 quote_nick_prefix = "<"
 quote_nick_suffix = ">"
-quote_time_format = "%H:%M:%S"
+quote_time_format = "%H:%M"
 read_marker = line
 read_marker_always_show = off
 read_marker_string = "- "
@@ -132,32 +135,32 @@
 window_auto_zoom = off
 window_separator_horizontal = on
 window_separator_vertical = on
-window_title = ""
+window_title = "Weechat ${info:version}"
 word_chars_highlight = "!\u00A0,-,_,|,alnum"
 word_chars_input = "!\u00A0,-,_,|,alnum"
 
 [palette]
 
 [color]
-bar_more = lightmagenta
+bar_more = 232
 chat = default
 chat_bg = default
 chat_buffer = white
 chat_channel = white
-chat_day_change = cyan
+chat_day_change = red
 chat_delimiters = green
-chat_highlight = yellow
-chat_highlight_bg = magenta
-chat_host = cyan
+chat_highlight = white
+chat_highlight_bg = 202
+chat_host = 244
 chat_inactive_buffer = default
 chat_inactive_window = default
-chat_nick = lightcyan
-chat_nick_colors = "cyan,magenta,green,brown,lightblue,default,lightcyan,lightmagenta,lightgreen,blue"
+chat_nick = magenta
+chat_nick_colors = "157,156,180,181,193,011,223,229,120,145,247"
 chat_nick_offline = default
 chat_nick_offline_highlight = default
 chat_nick_offline_highlight_bg = blue
-chat_nick_other = cyan
-chat_nick_prefix = green
+chat_nick_other = 91
+chat_nick_prefix = 69
 chat_nick_self = white
 chat_nick_suffix = green
 chat_prefix_action = white
@@ -177,31 +180,31 @@
 chat_text_found_bg = lightmagenta
 chat_time = default
 chat_time_delimiters = brown
-chat_value = cyan
+chat_value = 33
 chat_value_null = blue
 emphasized = yellow
 emphasized_bg = magenta
 input_actions = lightgreen
 input_text_not_found = red
 item_away = yellow
-nicklist_away = cyan
+nicklist_away = 33
 nicklist_group = green
-separator = blue
-status_count_highlight = magenta
+separator = green
+status_count_highlight = 200
 status_count_msg = brown
 status_count_other = default
 status_count_private = green
-status_data_highlight = lightmagenta
+status_data_highlight = 190
 status_data_msg = yellow
 status_data_other = default
-status_data_private = lightgreen
+status_data_private = 220
 status_filter = green
 status_more = yellow
-status_mouse = green
-status_name = white
-status_name_ssl = lightgreen
+status_mouse = 220
+status_name = 218
+status_name_ssl = 229
 status_nicklist_count = default
-status_number = yellow
+status_number = 187
 status_time = default
 
 [completion]
@@ -250,21 +253,21 @@
 buflist.filling_left_right = vertical
 buflist.filling_top_bottom = columns_vertical
 buflist.hidden = off
-buflist.items = ""
+buflist.items = "buflist"
 buflist.position = left
 buflist.priority = 0
 buflist.separator = on
 buflist.size = 0
-buflist.size_max = 0
+buflist.size_max = 18
 buflist.type = root
 fset.color_bg = default
-fset.color_delim = cyan
+fset.color_delim = default
 fset.color_fg = default
 fset.conditions = "${buffer.full_name} == fset.fset"
 fset.filling_left_right = vertical
 fset.filling_top_bottom = horizontal
 fset.hidden = off
-fset.items = ""
+fset.items = "fset"
 fset.position = top
 fset.priority = 0
 fset.separator = on
@@ -272,21 +275,35 @@
 fset.size_max = 3
 fset.type = window
 input.color_bg = default
-input.color_delim = cyan
-input.color_fg = default
+input.color_delim = default
+input.color_fg = 187
 input.conditions = ""
 input.filling_left_right = vertical
 input.filling_top_bottom = horizontal
 input.hidden = off
-input.items = "[input_prompt]+(away),[input_search],[input_paste],input_text"
+input.items = "[input.prompt],[input_search],[input_paste],input_text"
 input.position = bottom
 input.priority = 1000
 input.separator = off
 input.size = 1
 input.size_max = 0
 input.type = window
+isetbar.color_bg = default
+isetbar.color_delim = cyan
+isetbar.color_fg = 143
+isetbar.conditions = ""
+isetbar.filling_left_right = vertical
+isetbar.filling_top_bottom = horizontal
+isetbar.hidden = on
+isetbar.items = "isetbar_help"
+isetbar.position = top
+isetbar.priority = 0
+isetbar.separator = on
+isetbar.size = 3
+isetbar.size_max = 3
+isetbar.type = window
 nicklist.color_bg = default
-nicklist.color_delim = cyan
+nicklist.color_delim = red
 nicklist.color_fg = default
 nicklist.conditions = "${nicklist}"
 nicklist.filling_left_right = vertical
@@ -297,25 +314,25 @@
 nicklist.priority = 200
 nicklist.separator = on
 nicklist.size = 0
-nicklist.size_max = 0
+nicklist.size_max = 9
 nicklist.type = window
-status.color_bg = blue
-status.color_delim = cyan
-status.color_fg = default
+status.color_bg = 235
+status.color_delim = default
+status.color_fg = white
 status.conditions = ""
 status.filling_left_right = vertical
 status.filling_top_bottom = horizontal
 status.hidden = off
-status.items = "[time],[buffer_last_number],[buffer_plugin],buffer_number+:+buffer_name+(buffer_modes)+{buffer_nicklist_count}+buffer_zoom+buffer_filter,scroll,[lag],[hotlist],completion"
+status.items = "buffer_number+.,buffer_name +*buffer_nicklist_count*"
 status.position = bottom
 status.priority = 500
 status.separator = off
 status.size = 1
 status.size_max = 0
 status.type = window
-title.color_bg = blue
-title.color_delim = cyan
-title.color_fg = default
+title.color_bg = 235
+title.color_delim = red
+title.color_fg = 5
 title.conditions = ""
 title.filling_left_right = vertical
 title.filling_top_bottom = horizontal
@@ -329,6 +346,7 @@
 title.type = window
 
 [layout]
+_zoom.window = "1;0;0;0;irc;server.rizon"
 
 [notify]
 
@@ -362,8 +380,15 @@
 ctrl-W = "/input delete_previous_word"
 ctrl-X = "/input switch_active_buffer"
 ctrl-Y = "/input clipboard_paste"
+meta-ctrl-Q = "/window merge"
 meta-meta-OP = "/bar scroll buflist * b"
 meta-meta-OQ = "/bar scroll buflist * e"
+meta-meta-Oa = "/window up"
+meta-meta-Ob = "/window down"
+meta-meta-Or = "/window resize -1"
+meta-meta-Ox = "/window resize +1"
+meta-meta2-11~ = "/bar scroll buflist * b"
+meta-meta2-12~ = "/bar scroll buflist * e"
 meta-meta2-1~ = "/window scroll_top"
 meta-meta2-23~ = "/bar scroll nicklist * b"
 meta-meta2-24~ = "/bar scroll nicklist * e"
@@ -391,6 +416,7 @@
 meta-< = "/input jump_previously_visited_buffer"
 meta-= = "/filter toggle"
 meta-> = "/input jump_next_visited_buffer"
+meta-H = "/window splitv"
 meta-OA = "/input history_global_previous"
 meta-OB = "/input history_global_next"
 meta-OC = "/input move_next_word"
@@ -403,6 +429,11 @@
 meta-Ob = "/input history_global_next"
 meta-Oc = "/input move_next_word"
 meta-Od = "/input move_previous_word"
+meta-V = "/window splith"
+meta2-11^ = "/bar scroll buflist * -100%"
+meta2-11~ = "/bar scroll buflist * -100%"
+meta2-12^ = "/bar scroll buflist * +100%"
+meta2-12~ = "/bar scroll buflist * +100%"
 meta2-15~ = "/buffer -1"
 meta2-17~ = "/buffer +1"
 meta2-18~ = "/window -1"
@@ -413,10 +444,14 @@
 meta2-1;3D = "/buffer -1"
 meta2-1;3F = "/window scroll_bottom"
 meta2-1;3H = "/window scroll_top"
+meta2-1;3P = "/bar scroll buflist * b"
+meta2-1;3Q = "/bar scroll buflist * e"
 meta2-1;5A = "/input history_global_previous"
 meta2-1;5B = "/input history_global_next"
 meta2-1;5C = "/input move_next_word"
 meta2-1;5D = "/input move_previous_word"
+meta2-1;5P = "/bar scroll buflist * -100%"
+meta2-1;5Q = "/bar scroll buflist * +100%"
 meta2-1~ = "/input move_beginning_of_line"
 meta2-200~ = "/input paste_start"
 meta2-201~ = "/input paste_stop"
@@ -446,7 +481,7 @@
 meta2-[E = "/buffer -1"
 meta-_ = "/input redo"
 meta-a = "/input jump_smart"
-meta-b = "/input move_previous_word"
+meta-b = "/bar hide buflist"
 meta-d = "/input delete_next_word"
 meta-f = "/input move_next_word"
 meta-h = "/input hotlist_clear"
@@ -558,9 +593,11 @@
 meta-m = "/mute mouse toggle"
 meta-n = "/window scroll_next_highlight"
 meta-p = "/window scroll_previous_highlight"
+meta-q = "/bar hide buflist"
 meta-r = "/input delete_line"
 meta-s = "/mute aspell toggle"
 meta-u = "/window scroll_unread"
+meta-w = "/bar show buflist"
 meta-wmeta-meta2-A = "/window up"
 meta-wmeta-meta2-B = "/window down"
 meta-wmeta-meta2-C = "/window right"
@@ -621,6 +658,10 @@
 @chat(fset.fset):button2* = "hsignal:fset_mouse"
 @chat(fset.fset):wheeldown = "/fset -down 5"
 @chat(fset.fset):wheelup = "/fset -up 5"
+@chat(perl.iset):button1 = "hsignal:iset_mouse"
+@chat(perl.iset):button2* = "hsignal:iset_mouse"
+@chat(perl.iset):wheeldown = "/repeat 5 /iset **down"
+@chat(perl.iset):wheelup = "/repeat 5 /iset **up"
 @chat(script.scripts):button1 = "/window ${_window_number};/script go ${_chat_line_y}"
 @chat(script.scripts):button2 = "/window ${_window_number};/script go ${_chat_line_y};/script installremove -q ${script_name_with_extension}"
 @chat(script.scripts):wheeldown = "/script down 5"
